nifi-assembly/pom.xml

```diff
@@ -19,6 +19,10 @@ language governing permissions and limitations under the License. -->
     <artifactId>nifi-assembly</artifactId>
     <packaging>pom</packaging>
     <description>This is the assembly Apache NiFi</description>
+    <!--Add a blank run.as in case RPM isn't being used -->
+    <properties>
+        <nifi.run.as />
```

Comment:
@trixpan I guess what I'm saying is that with this change I think -Prpm makes a good rpm but not a good .tar.gz or .zip since the user will now need to edit bootstrap.conf if they're using the archive package; install.sh doesn't create the nifi user, right? I think somehow or other it needs to use the defaultUsername that the maven-rpm-plugin is using but I'm not sure how without digging further.